code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingConstants.java

```diff
@@ -31,7 +31,7 @@ public final class MessagingConstants {
     static final int DEFAULT_TIMEOUT = 5;
     static final long RESPONSE_CALLBACK_TIMEOUT = 10000; // 10 seconds
     static final long CACHE_EXPIRY_TIME = 604800000; // 7 days in milliseconds
-    static final int HALF_SECOND_IN_MILLIS = 500;
+    static final String DATA_STORE_NAME = "AdobeMobile_Messaging";
```

Comment:
It's common to use `DATA_STORE_NAME = EXTENSION_NAME`, unless you have multiple data stores.